File: Cookpot-Backend/cookpot-bl/DataModel/Recipe.cs
using System;
using System.Collections.Generic;

namespace cookpot.bl.DataModel
{
    public class Recipe
    {
        
        public int Id { get; set; }
        public Nullable<int> DurationTime { get; set; }
        
        public string DurationUnit { get; set; }
        
        public string RecipeName { get; set; }
        
        public List<Step> Steps { get; set; }
    }
}

File: Cookpot-Backend/cookpot-rest/ClientApp/src/classes/recipe.ts
import { Step } from './step';
import { Ingredient } from './ingredient';

export class Recipe {
  public Id: number;
  public DurationTime?: number;
  public DurationUnit?: string;
  public RecipeName: string = '';
  public Steps: Array<Step> = [];
  public Ingredients: Array<Ingredient> = [];
}